<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SshTunnelWidget</class>
 <widget class="QWidget" name="SshTunnelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSH tunnel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="use_ssh_tunnel">
     <property name="text">
      <string>Connect via SSH tunnel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ssh_groupBox">
     <property name="title">
      <string>SSH tunnel options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="use_loopback">
        <property name="text">
         <string>Tunnel via loopback address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port:</string>
        </property>
        <property name="buddy">
         <cstring>ssh_tunnel_port</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="ssh_tunnel_port">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>22</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User name:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="ssh_tunnel_user_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
